packages/forklift-console-plugin/src/modules/Overview/dynamic-plugin.ts

```diff
@@ -37,4 +38,14 @@ export const extensions: EncodedExtension[] = [
       required: ['CAN_LIST_NS'],
     },
   } as EncodedExtension<HrefNavItem>,
+
```

Comment:
Lint: we don't use empty lines between extentions

Reply:
Currently we do use empty lines between extensions in most of our dynamic-plugin files..
I'll fix here for the Overview file and will need to fix for others in a follow ups...